src/build.c

```diff
@@ -1547,25 +1542,17 @@ static void set_stop_button(gboolean stop)
 	GtkToolButton *run_button;
 
 	run_button = GTK_TOOL_BUTTON(toolbar_get_widget_by_name("Run"));
-	if (run_button != NULL)
-		button_stock_id = gtk_tool_button_get_stock_id(run_button);
+	if (!run_button) return;
```

Comment:
Minor nit, but should have space after `!` and line break before `return`.

Reply:
@codebrainz Is this documented in HACKING?

Reply:
More or less: [Line break](https://github.com/geany/geany/blob/0ce5d5484e39edc25bc996cc0bf01f5bdc2b5221/HACKING#L274), and [operator spacing](https://github.com/geany/geany/blob/0ce5d5484e39edc25bc996cc0bf01f5bdc2b5221/HACKING#L269) (although it says two operand) but also [fit in with existing code](https://github.com/geany/geany/blob/0ce5d5484e39edc25bc996cc0bf01f5bdc2b5221/HACKING#L278), for example [here](https://github.com/geany/geany/pull/2271/files#diff-8ae5a6e4e9368d91c0c59edbde1627c2R791) and [here](https://github.com/geany/geany/pull/2271/files#diff-8ae5a6e4e9368d91c0c59edbde1627c2R1550) in nearby code (counter example [here](https://github.com/geany/geany/pull/2271/files#diff-8ae5a6e4e9368d91c0c59edbde1627c2R830))

Reply:
I've added the line break to comply with HACKING. There are 3 existing if statements in build.c with a return clause on the same line, but I see that there are not many of these in `src/`.

> space after !

I haven't done this, there are 16 matches for the regex `\bif \(!\S` in build.c.